src/lib/sidebar/PipelineSchemeForm.svelte

```diff
@@ -101,95 +103,11 @@
       />
     </fieldset>
 
-    <fieldset class="govuk-fieldset">
-      <legend class="govuk-fieldset__legend">Timing and status</legend>
-
-      <Radio
-        legend="Status"
-        id="scheme-status"
-        choices={[
-          ["planned", "Planned"],
-          ["in development", "In development"],
-          ["in construction", "In construction"],
-          ["completed", "Completed"],
-        ]}
-        inlineSmall
-        required
-        bind:value={scheme.pipeline.status}
-      />
-
-      <Radio
-        legend="Timescale"
-        id="scheme-timescale"
-        choices={[
-          ["short", "Short (1-3 years)"],
-          ["medium", "Medium (3-6 years)"],
-          ["long", "Long (6-10 years)"],
-        ]}
-        inlineSmall
-        required
-        bind:value={scheme.pipeline.timescale}
-      />
-      <NumberInput
-        label="Estimated completion year (if known)"
-        width={4}
-        min={2010}
-        max={2100}
-        bind:value={scheme.pipeline.timescale_year}
-      />
-
-      <NumberInput
-        label="What year was this scheme most recently published?"
-        width={4}
-        min={2010}
-        max={2100}
-        bind:value={scheme.pipeline.year_published}
-      />
-
-      <NumberInput
-        label="What year was this scheme most recently consulted on?"
-        width={4}
-        min={2010}
-        max={2100}
-        bind:value={scheme.pipeline.year_consulted}
-      />
-    </fieldset>
-
-    <fieldset class="govuk-fieldset">
-      <legend class="govuk-fieldset__legend">Budget</legend>
-
-      <NumberInput
-        label="GBP funded"
-        width={10}
-        min={0}
-        bind:value={scheme.pipeline.budget_funded}
-      />
-      <NumberInput
-        label="GBP unfunded"
-        width={10}
-        min={0}
-        bind:value={scheme.pipeline.budget_unfunded}
-      />
-
-      <Radio
-        legend="Funding source"
-        id="scheme-funding-source"
-        choices={[
-          repeat("ATF2"),
-          repeat("ATF3"),
-          repeat("ATF4"),
-          repeat("ATF4E"),
-          repeat("CRSTS"),
-          repeat("LUF"),
-        ]}
-        inlineSmall
-        bind:value={scheme.pipeline.funding_source}
-      />
-
-      <Checkbox id="scheme-funded" bind:checked={scheme.pipeline.funded}>
-        Is the scheme fully funded?
-      </Checkbox>
-    </fieldset>
+    <TimescaleSubform
+      bind:timescale={scheme.pipeline.scheme_timescale}
+      maxTimescaleFromInterventions={maxTimescale}
+    />
+    <BudgetSubform bind:budget={scheme.pipeline.scheme_budget} {summedCost} />
```

Comment:
Similar comment here -- why is the parent component bothering with the summedCost and then passing it down? BudgetSubForm can just call a read-only helper function on $gjSchemeCollection and get back a hint to diplay